### PR TITLE
Add webpack:compile task to debian build proccess [deb]

### PR DESCRIPTION
**Pull request by tbrisker**
This is needed for theforeman/foreman#3433 to
build the webpack managed assets correctly for production.

### CODE REVIEW

**Comment by mmoll** [comment]
I didn't test this, but do I get it correctly that, as the result in a package different to now, there will be a public/webpack folder where the assets live in?

File: debian/jessie/foreman/rules [hunk 1]
```diff
@@ -19,6 +19,7 @@ build/foreman::
 	/usr/bin/gem install -f thor -v '0.19.1'
 	/usr/bin/bundle package
```
Comment (domcleal) [inline debian/jessie/foreman/rules]: Presumably an `npm install` is also required here?

Comment (tbrisker) [inline debian/jessie/foreman/rules]: Yes, thanks, I will update this.

**Comment by mmoll** [comment]
seems to do its job :)

**Comment by mmoll** [comment]
[test]

**Comment by mmoll** [comment]
Could you please add `nodejs-legacy` to the Builddeps? /usr/bin/node should be linked to /usr/bin/nodejs then and webpack should work.

**Comment by tbrisker** [comment]
@mmoll added.

**Comment by domcleal** [comment]
Merged as 6c4f1de, thanks @tbrisker.

**Pull request closed without merging**